src/main/java/de/tum/in/www1/artemis/service/FileService.java

```diff
@@ -294,10 +294,10 @@ public void copyResources(Resource[] resources, String prefix, String targetDire
 
         for (Resource resource : resources) {
 
-            String fileUrl = java.net.URLDecoder.decode(resource.getURL().toString(), "UTF-8");
+            String filePath = resource.getFile().getAbsolutePath();
```

Comment:
This does not solve the issue for windows as we then execute string operations on the path. As a result the escape character "\" and the following character are not counted as part of the string.